beacon-chain/operations/slashings/metrics.go

```diff
@@ -0,0 +1,45 @@
+package slashings
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
```

Comment:
can we follow prometheus's metric and label naming conventions? (ie. end with `_total` for most of these)

https://prometheus.io/docs/practices/naming/